src/test/java/friday/model/student/MasteryCheckTest.java

```diff
@@ -14,8 +15,23 @@ public void constructor_null_throwsNullPointerException() {
 
     @Test
     public void isValidMasteryCheck() {
+        assertFalse(MasteryCheck.isValidMasteryCheck("09-01-2022")); // wrong order
+        assertFalse(MasteryCheck.isValidMasteryCheck("20220901")); // missing '-' symbols
+        assertFalse(MasteryCheck.isValidMasteryCheck("01-01")); // missing year
+        assertFalse(MasteryCheck.isValidMasteryCheck("0000-12-05")); // invalid year
+        assertFalse(MasteryCheck.isValidMasteryCheck("2022-13-05")); // invalid month
+        assertFalse(MasteryCheck.isValidMasteryCheck("0000-12-32")); // invalid day
```

Comment:
"0000-12-32" seems to be checking both invalid year and invalid day. Would it be better to use "2022 -12-32" to check for invalid day